src/test/java/com/jcabi/dynamo/mock/H2DataTest.java

```diff
@@ -134,7 +134,28 @@ public void createsTablesWithLongNames() throws Exception {
      */
     @Test
     public void supportsTableNamesWithIllegalCharacters() throws Exception {
-        new H2Data().with(".-.", new String[] {"pk"}, new String[0]);
+        new H2Data().with(".-.", new String[]{"pk"}, new String[0]);
+    }
+
+    /**
+     * H2Data supports column names with characters illegal to H2.
+     * @throws Exception In case test fails
+     */
+    @Test
+    @Ignore
+    public void supportsColumnNamesWithIllegalCharacters() throws Exception {
+        /* todo: #28:30min H2Data doesn't support COLUMNS with
+         * ".", "-" or digits
+         * but i don't know for sure
+         * should it support these symbols or not.
+         * It's needed to be confirmed and test should be uncommented
+         * when H2Data will be supporting mentioned symbols.
+         * */
+        final String key = "0-.col.-0";
```

Comment:
can't do that
----------------
[ERROR] /src/test/java/com/jcabi/dynamo/mock/H2DataTest.java[152]: The String "0-.col.-0" appears 2 times in the file. (MultipleStringLiteralsCheck)